Document/0x05d-Testing-Data-Storage.md

```diff
@@ -796,6 +796,12 @@ After executing all available app functions, attempt to back up via `adb`. If th
 ```shell
 $ adb backup -apk -nosystem <package-name>
 ```
+ADB should respond now with "Now unlock your device and confirm the backup operation" and you should be asked on the Android phone for a password. This is an optional step and you don't need to provide one. If the phone does not prompt this message, try the following command including the quotes:
+
+```shell
+$ adb backup "-apk -nosystem <package-name>"
+```
+The problem happens when your device has an adb version prior to 1.0.31. If that's the case you must use an adb version of 1.0.31 also on hour host machine. Versions of adb after 1.0.32 [broke the backwards compatibility.](https://issuetracker.google.com/issues/37096097)
```

Comment:
hour -> your 

```
[broke the backwards compatibility.](https://issuetracker.google.com/issues/37096097) 

change to:

[broke the backwards compatibility.](https://issuetracker.google.com/issues/37096097 "adb backup is broken since ADB version 1.0.32")
```

Reply:
@sushi2k , I just adjusted!